portal-api/src/v1/users/sign-in-link.controller.js

```diff
@@ -44,8 +44,6 @@ class SignInLinkController {
           });
         }
         case SIGN_IN_LINK.STATUS.VALID: {
-          await this.#signInLinkService.resetSignInData(userId);
```

Comment:
this can be removed due to updated theupdateLastLogin to updateLastLoginAndResetSignInData.